### PR TITLE
Fall 2020/hypertext assassins

### PR DESCRIPTION
**Pull request by whymarlene**
Removed Foo and modified README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +0,0 @@
-![Bits of Good](/assets/header.png)
```
Comment (Arthelon) [inline README.md]: Please undo this file deletion

File: cohorts/1/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+    <head>
+        <title>README</title>
```
Comment (Arthelon) [inline cohorts/1/index.html]: Add a more descriptive page title, maybe your cohort name

**Review by Arthelon — CHANGES_REQUESTED** [review]
some small changes

File: README.md (hunk 1 above)
Comment (whymarlene) [reply README.md]: Is this the README for our cohort or the BoG README?

**Review by Arthelon — APPROVED** [review]

**Pull request merged**